kubernetes/services/dpm_manager.yaml

```diff
@@ -62,8 +62,8 @@ data:
     group.topic.to.multicast.topic.map=multicast-topic1:group-topic1,group-topic3 multicast-topic2:group-topic2,group-topic4
 
 
-    arionGateway.enabled = false
-    arionMaster.server = 127.0.0.1
+    arionGateway.enabled = true
```

Comment:
I assume we should leave it 'false' by default? To not break legacy E2E testing

Reply:
@lfu-ps you are correct, thanks.